### PR TITLE
add monitoring section

### PR DESCRIPTION
**Pull request by aviramha**
Pending operator release + chart release.

### CODE REVIEW

File: content/docs/managing-mirrord/monitoring/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+weight: 500
+toc: true
+tags: ["team", "enterprise"]
+---
+
+## Monitoring
+
+mirrord Operator can produce logs in JSON format to be digested by most logging solutions (DataDog, Dynatrace, etc).
+To enable JSON logging, specify `json_log: true` under `operator` in the operator helm chart values.
+Log level is info by default, and can be altered via operator deployment env using `RUST_LOG` (directive)[https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html].
```
Comment (Razz4780) [inline content/docs/managing-mirrord/monitoring/index.md:21]:
```suggestion
Log level is info by default, and can be altered via operator container environment using `RUST_LOG` [directive](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html).
```

**Review by Razz4780 — CHANGES_REQUESTED** [review]

File: content/docs/managing-mirrord/monitoring/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+teams:
+weight: 500
+toc: true
+tags: ["team", "enterprise"]
+---
+
+## Monitoring
+
+mirrord Operator can produce logs in JSON format to be digested by most logging solutions (DataDog, Dynatrace, etc).
+To enable JSON logging, specify `json_log: true` under `operator` in the operator helm chart values.
```
[the remark this answers is not in the document]
Comment (aviramha) [reply content/docs/managing-mirrord/monitoring/index.md:20]: No, we don't support any complex configuration in the setup.

**Review by Razz4780 — APPROVED** [review]

File: content/docs/managing-mirrord/monitoring/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+weight: 500
+toc: true
+tags: ["team", "enterprise"]
+---
+
+## Monitoring
+
+The mirrord Operator can produce logs in JSON format that can be digested by most popular logging solutions (DataDog, Dynatrace, etc).
+To enable JSON logging, set `operator.json_log` to `true` in the Operator Helm chart values.
+Log level is info by default, and can be altered via operator container environment using `RUST_LOG` [directive](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html).
```
Comment (eyalb181) [inline content/docs/managing-mirrord/monitoring/index.md:21]:
```suggestion
The log level is `INFO` by default, and can be changed via operator container environment using the `RUST_LOG` [directive](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html).
```

Comment (eyalb181) [reply content/docs/managing-mirrord/monitoring/index.md:21]: You're sending the reader to do a Rust PhD with that link, could you just put an example instead?

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/docs/managing-mirrord/monitoring/index.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+weight: 500
+toc: true
+tags: ["team", "enterprise"]
+---
+
+## Monitoring
+
+The mirrord Operator can produce logs in JSON format that can be digested by most popular logging solutions (DataDog, Dynatrace, etc).
+To enable JSON logging, set `operator.jsonLog` to `true` in the Operator Helm chart values.
+The log level is `INFO` by default, and can be changed via operator container environment using the `RUST_LOG` [directive](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html). The most common value would be `mirrord=trace`/`mirrord=debug` etc.
```
Comment (eyalb181) [inline content/docs/managing-mirrord/monitoring/index.md:21]:
```suggestion
The log level is `INFO` by default, and can be changed using the `RUST_LOG` environment variable in the Operator container, which takes values in the following format: `mirrord={log_level}` (e.g. `mirrord=debug`).
```

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**